static/js/popovers.js

```diff
@@ -150,8 +150,10 @@ exports.toggle_actions_popover = function (element, id) {
         var editability = message_edit.get_editability(message);
         var use_edit_icon;
         var editability_menu_item;
+        var delete_limit_if_apply = false;
```

Comment:
This is sort of an awkward variable name.  I don't know exactly what it means.  The `if_apply` is confusing for a boolean.

Reply:
This is used to check that if admin has set the time limit for editing a message or not. 
If time limit for editing is 
**set**: same time limit will also set for deleting
**not set**: infinite time limit for deleting(no time limit)
What do you suggest a name for this variable?